docs/website/blog/2020-01-06-release-0.5.2.md

```diff
@@ -0,0 +1,168 @@
+- [#2712](https://github.com/onivim/oni2/pull/2712): Add support for additionalTextEdits (auto-import) for completion provider
+
+### Vim fixes
+
+Of particular note on the Vim-side, I've been starting to do work to unblock
+Vim extensions. These expose some bugs in [`libvim`](https://github.com/onivim/libvim) that I've been addressing.... but we're getting closer to having a plugin like vim-surround working:
+
+![2021-01-06 14 41 39](https://user-images.githubusercontent.com/13532591/103827241-68fb9280-502d-11eb-8c22-057354dff8be.gif)
+
+(Note that the keydisplayer doesn't pick up `getchar()` right now - the key sequence I pressed is `cs(}`)
+
+This isn't ready for prime-time yet, and still lots of blockers - but the gist is that we are making forward progress here, too.
+
+Other notable fixes:
+
+- [#2351](https://github.com/onivim/oni2/pull/2351): Fix crash on `:enew`
+- [#2373](https://github.com/onivim/oni2/pull/2373): Macro indicator in status bar
+- [#2364](https://github.com/onivim/oni2/pull/2364): Show pending operator in status bar
+- [#2429](https://github.com/onivim/oni2/pull/2429): Fix visual-in motion (viw/vi") commands
+- [#2574](https://github.com/onivim/oni2/pull/2574): Flakiness in cursor positioning after leaving insert mode
+- [#2699](https://github.com/onivim/oni2/pull/2699): Implement `getchar()` handler (needed for VimL plugin support)
+- [#2765](https://github.com/onivim/oni2/pull/2765): Hook up `messages` and `messages clear` ex commands
+- [#2792](https://github.com/onivim/oni2/pull/2792): Handle count for insert mode commands
+
+This is just a snapshot of the changes in 0.5.2 and 0.5.1. The full
+changelogs are available here:
+
+- [0.5.1 Changelog](https://github.com/onivim/oni2/blob/66f31792d6f650f044a8d22861c82f78ff142b52/CHANGES_CURRENT.md)
+- [0.5.2 Changelog](https://github.com/onivim/oni2/blob/5c77004c930bfd4233418f1b7d8235447e1d7f12/CHANGES_CURRENT.md)
+
+One resolution I have for 2021 is more frequent updates - I've had to push deadlines back a few times and have longer gaps between updates - the downside with that is, often, there are smaller changes that would be nice to showcase... but fall through the cracks. I'd like to highlight monthly progress.
+
+I'm still working towards the 0.6.0 release - there's still a lot of work left - so these incoming, monthly `0.5.x` releases will be focusing on getting us to that milestone.
+
+Originally, the 0.6.0 release was planned for end of December - so unfortunately I will push it out, as there is still work left to accomplish these renaming goals:
+- VimL plugin integration
+- Snippet support
+- Vim code folding integration
+- Code lens
+- Code actions
+
+I'm going to switch the deadline to "When It's Done" - what I found in 2020
+was that I'd push aggressive deadlines for myself. I believe these deadline / estimates would be in valid in a vaccuum, ie, if I was 100% focused on coding justthose features... but, of course, that's simply never the reality.
```

Comment:
`vaccumm` -> `vacuum`, `ie` -> `i.e.`, `justthose` -> `just those`.

Reply:
Ah, thanks - I realized I deleted my spell checker while testing! Re-installing and taking a pass..